docs/quickstart.md

```diff
@@ -2,198 +2,441 @@
+>>> import pynapple as nap
+
+>>> path = nmo.fetch.fetch_data("A2929-200711.nwb")
+>>> data = nap.load_file(path)
+
+>>> # load spikes and head direction
+>>> spikes = data["units"]
+>>> head_dir = data["ry"]
 
-spikes = data["units"]
-head_dir = data["ry"]
+>>> # restrict and bin
+>>> counts = spikes[6].count(0.01, ep=head_dir.time_support)
 
-counts = spikes[6].count(0.01, ep=head_dir.time_support)  # restrict and bin
-upsampled_head_dir = head_dir.bin_average(0.01)  # up-sample head direction
+>>> # down-sample head direction
+>>> upsampled_head_dir = head_dir.bin_average(0.01)  
 
-# create your features
-X = nmo.basis.CyclicBSplineBasis(10).compute_features(upsampled_head_dir)
+>>> # create your features
+>>> X = nmo.basis.CyclicBSplineBasis(10).compute_features(upsampled_head_dir)
+
+>>> # add a neuron axis and fit model
+>>> model = nmo.glm.GLM().fit(X, counts) 
 
-# add a neuron axis and fit model
-model = nmo.glm.GLM().fit(X, counts) 
 ```
 
+
 Finally, let's compare the tuning curves
 
+
 ```python
-import numpy as np
-import matplotlib.pyplot as plt
 
-raw_tuning = nap.compute_1d_tuning_curves(spikes, head_dir, nb_bins=100)[6]
-model_tuning = nap.compute_1d_tuning_curves_continuous(
-    model.predict(X)[:, np.newaxis] * X.rate,  # scale by the sampling rate
-    head_dir,
-    nb_bins=100
-)[0]
+>>> import numpy as np
+>>> import matplotlib.pyplot as plt
+
+>>> # tuning curves
+>>> raw_tuning = nap.compute_1d_tuning_curves(spikes, head_dir, nb_bins=100)[6]
+
+>>> # model based tuning curve
+>>> model_tuning = nap.compute_1d_tuning_curves_continuous(
+...     model.predict(X)[:, np.newaxis] * X.rate,  # scale by the sampling rate
+...     head_dir,
+...     nb_bins=100
+...  )[0]
+
+
+>>> # plot results
+>>> sub = plt.subplot(111, projection="polar")
+>>> plt1 = plt.plot(raw_tuning.index, raw_tuning.values, label="raw")
+>>> plt2 = plt.plot(model_tuning.index, model_tuning.values, label="glm")
+>>> legend = plt.yticks([])
+>>> xlab = plt.xlabel("heading angle")
 
-# plot results
-plt.subplot(111, projection="polar")
-plt.plot(raw_tuning.index, raw_tuning.values, label="raw")
-plt.plot(model_tuning.index, model_tuning.values, label="glm")
-plt.legend()
-plt.yticks([])
-plt.xlabel("heading angle")
-plt.show()
 ```
 
-![Alt text](head_dir_tuning.jpg)
 
-### Compatibility with `scikit-learn`
+<img src="../head_dir_tuning.jpg" width="40%">
+
+
+## __Compatibility with `scikit-learn`__
+
 
 `scikit-learn` is a machine learning toolkit that offers advanced features like pipelines and cross-validation methods. 
+NeMoS takes advantage of these features, while still gaining the benefit of JAX's just-in-time compilation and GPU-acceleration!
 
-NeMoS takes advantage of these features, while still gaining the benefit of JAX's just-in-time 
-compilation and GPU-acceleration!
+For example, if we would like to tune the critical hyper-parameter `regularizer_strength`, we  could easily run a `K-Fold` cross-validation using `scikit-learn`.
```

Comment:
link to somewhere (probably scikit learn docs) for more info on cross-validation? this doesn't tell someone what cross-validation is

Reply:
should also probably have a brief description of what and why cross-validation here

Reply:
I agree for what it concerns the link and pointing to additional refs (we have already a link to our own tutorial on cv and pipelining, but adding extra refs is a good idea). 

For the second point, I disagree, I would avoid an extra explanation here;

I think the quickstart should be streamlined, how to guide style to speak in diataxis terms, while  the k-fold explenation would belongs in a background type of page.